LEAF_Request_Portal/admin/index.php

```diff
@@ -422,9 +422,7 @@ function customTemplate($tpl)
             $t_form->assign('orgchartPath', Config::$orgchartPath);
             $t_form->assign('CSRFToken', $_SESSION['CSRFToken']);
 
-            $main->assign('javascripts', array('../../libs/js/jquery/jquery.min.js',
-                                           '../../libs/js/jquery/jquery-ui.custom.min.js',
-                                           '../../libs/js/jsPlumb/dom.jsPlumb-min.js', ));
+            $main->assign('javascripts', array('../../libs/js/jsPlumb/dom.jsPlumb-min.js', ));
```

Comment:
Not sure why this is included here and in main.tpl

Reply:
it is causing issues because jQuery is included twice

Reply:
Where else is it included?

Reply:
main.tpl